curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-a-expense-tracker/657e4e9014f900059633832b.md

```diff
@@ -0,0 +1,42 @@
+---
+id: 657e4e9014f900059633832b
+title: Step 3
+challengeType: 20
+dashedName: step-3
+---
+
+# --description--
+
+Functions in Python are like mini-programs within your program. They perform a specific task and can be used repeatedly throughout your code. To define a function, use the `def` keyword, followed by the function name and parentheses `()`.
+
+Begin by defining a function called `add_expense`. This function will take three arguments, `expenses`, `amount` and `category`. Inside the function body, include a `pass` statement. The pass statement is used as a placeholder in Python, indicating that the function currently does nothing. It's a way to define the function structure without adding any operational code yet.
+
+This is the syntax of a function with a pass statement:
+```py
+def function_name():
+    pass
+```
+    
+
+
+# --hints--
+
+The function should be named `add_expense` and it should have 3 arguments.
+
+```js
+({
+  test: () => assert.match(code, /def\s+add_expense\s*\(\s*expenses\s*,\s*amount\s*,\s*category\s*\)\s*:/)
+})
+
+```
+
+
+# --seed--
+
+## --seed-contents--
+
+```py
+--fcc-editable-region--
+# Define the 'add_expense' function below
```

Comment:
Should the `single_expense` dictionary defined in the previous step be here? The description did not mention that the dictionary should be removed.

Reply:
I have added a line in Step 2 that talks about this.